source/validate_id.cpp

```diff
@@ -264,6 +264,30 @@ bool idUsage::isValid<SpvOpTypeSampler>(const spv_instruction_t*,
   return true;
 }
 
+// True if the integer constant is > 0. constWords are words of the
+// constant-defining instruction (either OpConstant or
+// OpSpecConstant). typeWords are the words of the constant's-type-defining
+// OpTypeInt.
+bool aboveZero(const std::vector<uint32_t>& constWords,
+               const std::vector<uint32_t>& typeWords) {
+  const auto width = typeWords[2];
+  const bool is_signed = typeWords[3];
+  if (width == 64) {
```

Comment:
The SPIR-V spec doesn't allow 48-bit integers -- the "Data rules" part of section 2.16.1 allows only 32-bit integers plus whatever's enabled by capabilities.  And capabilities only exist for Int64, Int16, and Int8.